src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -1096,11 +1096,11 @@ static ITypeSymbol MakeNullable(ITypeSymbol symbol, Compilation compilation)
                     }
                     else if (symbol.IsValueType)
                     {
-                        return compilation.GetSpecialType(SpecialType.System_Nullable_T).Construct(symbol.WithoutNullability());
+                        return compilation.GetSpecialType(SpecialType.System_Nullable_T).Construct(symbol);
```

Comment:
For curiosity, why is this change correct?

Reply:
So if we're making a value type nullable, the value type shouldn't already have a top-level nullability on it in the first place, because it's a value type. The WithoutNullability() would have been required here to strip the wrapper as else Construct would have thrown.

Good question though -- some WithoutNullabilities() were being used for actually stripping an annotation that might exist, and some existed to make the existing compiler APIs not throw. I'm thinking this is safe in the latter category.